db/models/Map.js

```diff
@@ -11,6 +11,15 @@ const MapSchema = new Schema({
 	edges: String,
 });
 
+const UserSchema = new Schema({
```

Comment:
I guess it's fine, but we normally use a separate folder + file for different schemas.